src/gutenberg2zim/download.py

```diff
@@ -73,12 +72,12 @@ def is_safe(fname):
     # move all extracted files to proper locations
     for zipped_file in zipped_files:
         # skip folders
-        if not Path(zipped_file).ext:
+        if not Path(zipped_file).suffix:
             continue
 
-        src = os.path.join(tmpd, zipped_file)
-        if os.path.exists(src):
-            fname = Path(zipped_file).basename()
+        src = Path(tmpd) / zipped_file
+        if Path(src).exists():
+            fname = Path(zipped_file).name
```

Comment:
`src` is already a Path
```suggestion
        if src.exists():
```